src/Backend/Core/Engine/Base/Config.php

```diff
@@ -136,20 +147,36 @@ private function isActionDisabled(string $actionType, string $action): bool
                 return in_array($action, $this->disabledAJAXActions, true);
         }
 
-        throw new \Exception($actionType . ' is not a valid action type');
+        throw new InvalidArgumentException($actionType . ' is not a valid action type');
```

Comment:
I would keep it in since it is used by some public methods like `isActionAvailableForActionType` and `getActionClass`